docker-compose.golang.yml

```diff
@@ -34,14 +38,7 @@ services:
         VERSION: "1.21"
         # Need to override this because default value doesn't work with this golang version
         CONTROLLER_GEN_VERSION: "v0.14.0"
-
```

Comment:
removed go1.20 , it's two years old, and most of the tools we use don't support it.